### PR TITLE
Update zh_cn (1.18)

### PR DESCRIPTION
**Pull request by Cactusstudent**

### CODE REVIEW

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/boots.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "properties": [
     "390 耐久度",
     "+2 护甲值",
+    "+2 Toughness",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/boots.json]:
```suggestion
    "+2 盔甲韧性",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/chestplate.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
   "properties": [
     "480 耐久度",
     "+7 护甲值",
+    "+2 Toughness",
+    "40% Unarmed Attack",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/chestplate.json]:
```suggestion
    "+2 盔甲韧性",
    "40% 赤手空拳伤害",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/helmet.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   "properties": [
     "330 耐久度",
     "+2 护甲值",
+    "+2 Toughness",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/helmet.json]:
```suggestion
    "+2 盔甲韧性",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/leggings.json [hunk 4]
```diff
@@ -6,6 +6,7 @@
   "properties": [
     "450 耐久度",
     "+5 护甲值",
+    "+2 Toughness",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/plate/leggings.json]:
```suggestion
    "+2 盔甲韧性",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/slime/slimelytra.json [hunk 5]
```diff
@@ -6,6 +6,7 @@
   ],
   "properties": [
     "672 耐久度",
+    "40% Unarmed Attack",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/slime/slimelytra.json]:
```suggestion
    "40% 赤手空拳伤害",
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_ability.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Extract Ability",
+  "image": {
+    "file": "tconstruct:book/images/worktable/extract_ability.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "Extracting ability modifiers from enchanted tools is a quite expensive process, requiring not only ichor crystals but also a diamond to help the process. Even with the power of ichor, the tool is still lost, seems you need enderslime if you wish to preserve the tool."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_ability.json]:
```suggestion
  "title": "能力强化提取",
  "image": {
    "file": "tconstruct:book/images/worktable/extract_ability.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "从附魔工具上提取能力强化的过程相当费材料，不止需要灵浆水晶，还需要钻石来辅助提取。就算有灵浆的力量支持，工具还是没有办法保留下来，似乎只有末影黏液才能让你保住工具了。"
```
may have many colloquial word choices, do accept suggestion with caution

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_defense.json [hunk 7]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Extract Defense",
+  "image": {
+    "file": "tconstruct:book/images/worktable/extract_defense.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "Extracting defense modifiers from enchanted books or tools requires earthslime crystals and gold. The process is similarly unstable casuing loss of the tool."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_defense.json]:
```suggestion
  "title": "防御能力提取",
  "image": {
    "file": "tconstruct:book/images/worktable/extract_defense.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "从附魔书或附魔工具上提取防御能力强化需要末影黏液水晶和金。这一过程同样较不稳定，也会导致工具的损失。"
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_modifier.json [hunk 8]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Extract Modifiers",
+  "image": {
+    "file": "tconstruct:book/images/worktable/extract_modifier.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "While it is not possible to preserve the items used to apply a modifier, I have managed to store the essence of the modifier into a crystal while removing it, allowing me to put that modifier on a different tool. The process requires an enderslime crystal and dragons breath to perform, so only worth it for expensive modifiers. This technique also works on enchanted tools as books, though the process removes all enchantments instead of just 1."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_modifier.json]:
```suggestion
  "title": "强化提取",
  "image": {
    "file": "tconstruct:book/images/worktable/extract_modifier.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "尽管不可能保留用于加持强化的物品，但我找到了能将强化的精华提出并放入水晶的方法，方便重新利用。这一过程需要末影黏液水晶和龙息，所以只有相当贵重的强化才适合这种提取过程。这种提取方法也适用于附魔工具和附魔书，不过和强化只移除一个不同，所有魔咒会被同时移除。"
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_upgrade.json [hunk 9]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Extract Defense",
+  "image": {
+    "file": "tconstruct:book/images/worktable/extract_upgrade.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "This discovery lead me to further experiment with other crystals and enchantments. It turns out that you can extract upgrade modifiers from enchanted books or tools using just skyslime crystals and lapis. The process seems a lot less stable than the enderslime method however, I have been unable to recover the tool."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_upgrade.json]:
```suggestion
  "title": "升级强化提取",
  "image": {
    "file": "tconstruct:book/images/worktable/extract_upgrade.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "在做出重要发现之后，我又进一步试验了其他水晶和魔咒的效果。升级强化可以只用碧空黏液水晶和青金石从附魔书或附魔工具上提取。这一过程相较末影水晶的提取过程而言更不稳定，工具本身也损坏得修复不了了。"
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/invisible_ink.json [hunk 10]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Invisible Ink",
+  "image": {
+    "file": "tconstruct:book/images/worktable/invisible_ink.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "I once accidentally spilled an invisibility potion on my tool, and discovered the potion actually works to hide modifiers from view. The technique does not seem that useful unless a modifier does not go with the tools aesthetic, but maybe a more artistically minded Tinker will find a use for it."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/invisible_ink.json]:
```suggestion
  "title": "隐形墨水",
  "image": {
    "file": "tconstruct:book/images/worktable/invisible_ink.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "我曾不小心把隐身药水撒到工具上过，但也发现这种药水能使强化“隐身”。这种隐形方法似乎除了让不够美观的强化隐形之外没有太大用处，但也许更有艺术品位的工匠能发现它新的用途吧。"
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/modifier_worktable.json [hunk 11]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Modifier Worktable",
+  "recipe": "tconstruct:tables/modifier_worktable",
+  "description": [
+    { "text": "Arranging seared stone like this makes it easier to target a specific modifier in the recipes." }
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/modifier_worktable.json]:
```suggestion
  "title": "强化调配台",
  "recipe": "tconstruct:tables/modifier_worktable",
  "description": [
    { "text": "按此方式排列焦黑石以简化选取特定强化的过程。" }
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/sort_modifiers.json [hunk 12]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Sort Modifiers",
+  "image": {
+    "file": "tconstruct:book/images/worktable/sort_modifiers.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "While adding a compass to tools magnetizes the tool, just holding the compass near the tool has an interesting effect on modifiers. You can actually use the compass to rearrange modifiers without removing them from the tool! Unlike invisible ink, this has more applications than art, as it seems conflicts in modifiers is resolved by their position on the tool."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/sort_modifiers.json]:
```suggestion
  "title": "强化排序",
  "image": {
    "file": "tconstruct:book/images/worktable/sort_modifiers.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "给工具加持一个指南针就能使其磁化，但在工具旁边放下指南针则会对强化有些特殊效果。指南针能用来给工具上的强化重新排序，并且不会移除它们！和隐形墨水不一样，这一过程的实际效用远不止美观层面，因为某些强化间的冲突可经由改变它们在工具上的位置解决。"
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/visible_ink.json [hunk 13]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Visible Ink",
+  "image": {
+    "file": "tconstruct:book/images/worktable/visible_ink.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "After some failure using sponges to make the modifier visible again, I remembered milk is effective at neutralizing potions. Sure enough, adding a bit of milk is all it takes to make the modifier once again visible."
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/visible_ink.json]:
```suggestion
  "title": "显形墨水",
  "image": {
    "file": "tconstruct:book/images/worktable/visible_ink.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "在若干次用海绵让强化显形的失败尝试后，我猛然想起奶在中和药水方面极为有效。实际也确实如此，只需要一点点奶就能让强化重新可见。"
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/travelers/vest.json [hunk 14]
```diff
@@ -6,9 +6,10 @@
   "properties": [
     "160 耐久度",
     "+5 护甲值",
+    "55% Unarmed Attack",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/armor/travelers/vest.json]:
```suggestion
    "55% 赤手空拳伤害",
```

File: src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_upgrade.json (hunk 9 above)
Comment (Cactusstudent) [reply src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_upgrade.json]: 提出 -> 提取？

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/small/dagger.json [hunk 15]
```diff
@@ -1,7 +1,7 @@
 {
   "tool": "tconstruct:dagger",
   "text": [
-    { "text": "匕首是一种快速而精准的武器，主副手均可使用。" },
+    { "text": "匕首是一种快速而精准的武器，主副手均可使用。Identical daggers can be stacked to 2 when modifying they are undamaged, providing a discount on modifiers." },
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/small/dagger.json]:
```suggestion
    { "text": "匕首是一种快速而精准的武器，主副手均可使用。完全一致且未损失耐久度的匕首在加持强化时的堆叠数为2，加持所需的材料的消耗量也会减少。" },
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/tools/dagger.json [hunk 16]
```diff
@@ -11,6 +11,7 @@
   "properties": [
     "低伤害",
     "高攻速",
-    "可以副手使用"
+    "可以副手使用",
+    "Stacks to 2 when modifying"
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/tools/dagger.json]:
```suggestion
    "加持强化时堆叠数为2"
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 17]
```diff
@@ -2428,6 +2427,11 @@
   "modifier.tconstruct.the_one_probe.flavor": "科学化！",
   "modifier.tconstruct.the_one_probe.description": "当手持或者附加于头盔时，能够显示指向方块的信息",
 
+  "_comment": "Internal modifiers",
+  "modifier.tconstruct.fast_use_item": "Fast Use Item",
+  "modifier.tconstruct.overslime_friend": "Overslime Friend",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "modifier.tconstruct.fast_use_item": "快速使用",
  "modifier.tconstruct.overslime_friend": "黏液之友",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 18]
```diff
@@ -2626,6 +2630,7 @@
   "recipe.tconstruct.modifier.not_enough_slot": "至少需要1个%s槽来添加该强化物",
   "recipe.tconstruct.modifier.not_enough_slots": "至少需要%s个%s槽来添加该强化物",
   "recipe.tconstruct.modifier.validate_slots": "这项改动留给工具的%s槽过少",
+  "recipe.tconstruct.modifier.min_level": "This recipe requires the tool to have at least %s level %s to apply",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "recipe.tconstruct.modifier.min_level": "此配方需要至少%s级%s才可使用",
```
Haven't yet tested this part of translations, may need to change the order of formatters.

**Comment by Cactusstudent** [comment]
Finished

**Comment by KnightMiner** [comment]
Just a heads up, planning to release an update in the next few days. There will be an additional update in the near future to finish some slimestaff features later, so let me know if you think this PR is in a state that it can be merged, or if you'd rather wait for that next release.

**Comment by Cactusstudent** [comment]
> Just a heads up, planning to release an update in the next few days. There will be an additional update in the near future to finish some slimestaff features later, so let me know if you think this PR is in a state that it can be merged, or if you'd rather wait for that next release.

Thanks, and Sorry to be late. We have translated all of the text. (But before merging, we need review some of the text, maybe it will take us.. 12 Hours? I'm not sure, but i believe we can finish before tomorrow afternoon [UTC+8]) So, feel free to merge this PR after that time. 

> And, ChuijkYahus will Update zh_cn next time, we are in the same team.

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 19]
```diff
@@ -2541,7 +2541,7 @@
   "tool_stat.tconstruct.block_amount": "格挡值：",
   "tool_stat.tconstruct.block_amount.description": "每次受到攻击时所阻挡伤害的最大值",
   "tool_stat.tconstruct.block_angle": "格挡角度：",
-  "tool_stat.tconstruct.block_angle.description": "此盾在其正前方两侧能提供保护的角度范围，超出该角度范围的攻击不会被阻挡",
+  "tool_stat.tconstruct.block_angle.description": "Total angle of protection in front when shielding out of a max 360 degrees. 超出该角度范围的攻击不会被阻挡",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "tool_stat.tconstruct.block_angle.description": "防御时前方能格挡攻击的总角度，超出该角度范围的攻击不会被阻挡",
```

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 20]
```diff
@@ -1489,20 +1499,20 @@
   "__comment": "Tier 3 Materials",
   "material.tconstruct.slimesteel": "黏钢",
   "material.tconstruct.slimesteel.flavor": "你要把钢黏到一起才能获得黏钢",
-  "material.tconstruct.slimesteel.encyclopedia": "增加25点黏液覆层，每级增加最大值的50%。",
+  "material.tconstruct.slimesteel.encyclopedia": "黏液覆层+25，每级增加最大值的50%。",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json]:
```suggestion
  "material.tconstruct.slimesteel.encyclopedia": "黏液覆层+25点，每级增加最大值的50%。",
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/abilities/ranged/crystalshot.json [hunk 21]
```diff
@@ -5,12 +5,12 @@
       "text": "弓不再消耗物品栏内箭矢，而是自己产生水晶镞。"
     },
     {
-      "text": "每发射一根箭额外消耗4点耐久度。"
+      "text": "每发射一根箭额外耐久度-4。"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/abilities/ranged/crystalshot.json]:
```suggestion
      "text": "每发射一根箭额外消耗4点耐久度。"
```

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/slotless/general/overslime.json [hunk 22]
```diff
@@ -13,6 +13,6 @@
   "effects": [
     "多次使用",
     "无需槽位",
-    "非黏液工具：-10% 采掘速度和攻击力，-0.5 护甲值"
+    "非黏液工具：-10%采掘速度和攻击力，-0.5护甲值"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/slotless/general/overslime.json]:
```suggestion
    "非黏液工具：采掘速度和攻击力-10%，护甲值-0.5"
```

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor.json [hunk 23]
```diff
@@ -5,6 +5,7 @@
       { "text": "\n- 头盔能通过按键控制特殊用途。" },
       { "text": "\n- 胸甲接受空手时起效的能力。多数已在交 互能力中介绍。" },
       { "text": "\n- 护腿接受物品栏相关的能力与为玩家提供 增益的被动能力。" },
-      { "text": "\n- 靴子接受移动与跳跃相关的能力。" }
+      { "text": "\n- 靴子接受移动与跳跃相关的能力。" },
+      { "text": "\n- Shields have special abilities on blocking, including stronger counterattack modifiers." }
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor.json]: Mark as NOT TRANS

Comment (ChuijkYahus) [reply src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor.json]:
```suggestion
      { "text": "\n- 盾牌拥有特殊格挡能力，包括更为强大的 反击强化。" }
```

**Comment by Cactusstudent** [comment]
@KnightMiner Have done. Thanks for waiting us. (And sorry to ping you)

**Pull request merged**